packages/babel-plugin-transform-exponentiation-operator/test/fixtures/regression/4403/expected.js

```diff
@@ -1,3 +1,4 @@
+var _ref;
+
 var a, b;
-var _ref = `${b++}`;
-a[_ref] = Math.pow(a[_ref], 1)
```

Comment:
This is actually an `AssignmentExpression`, not a `ExpressionStatement`. This is the bug.